README.md

```diff
@@ -2,30 +2,280 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Use the JOIN operator to combine rows from multiple tables.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+This lab will provide practice with writing SQL SELECT statements that use the JOIN operator to
+combine rows from multiple tables based on foreign key constraints.
+
+The database holds information for hockey teams, players, and divisions.
+
+- A division has several teams.  Each team is in one division.
+- A team has several players.  Each player plays for one team.
+
+![lab2 erd](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-sql-lab2/lab2_erd.png
+)
 
 ## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+1. Create a new database named `lab2`.
+2. Select the `lab2` database and open the query tool.          
+3. Open the `lab_setup.sql` file in the query tool, then execute the SQL statements to create
+   the tables and insert rows.
+4. Close the current connection by closing the tab on the query tool toolbar, and then open a new query
+   tool connection (to avoid overwriting lab_setup.sql).    
+5. Execute the following queries and confirm the tables are populated:   
+
+<table>
+<tr>
+<th>
+Query
+</th>
+<th>
+Result
+</th>
+</tr>
+
+<tr>
+<td>
+
+```sql
+SELECT *
+FROM division;
+```
+
+</td>
+
+<td>
+
+```text
+id  name
+
+1   Atlantic
+2   Central
+3   Northeast
+4   Northwest
+5   Southeast
+6   Pacific
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+```sql
+SELECT *
+FROM team;
+```
+
+</td>
+
+<td>
+
+```text
+id  name                    wins    losses  ties
+1   Boston Blue Jays        43	    16	    3
+2   Columbus Cardinals      4	    27	    2
```

Comment:
I got 34 wins? I just looked again at the lab_setup.sql, I believe this should be 34 wins.

Reply:
yes, thanks!